catalog/security/security-app/src/main/resources/features.xml

```diff
@@ -32,6 +32,12 @@
         <bundle>mvn:org.codice.alliance.catalog.plugin/catalog-plugin-defaultsecurityattributevalues/${project.version}</bundle>
     </feature>
 
+    <feature name="security-auditcontrolledaccess" version="${project.version}"
+      description="Plugin to audit controlled access.">
+        <feature>alliance-catalog-core-api</feature>
+        <bundle>mvn:org.codice.alliance.catalog.plugin/catalog-plugin-auditcontrolledaccess/${project.version}</bundle>
+    </feature>
+
```

Comment:
❓ How is this feature being loaded into the environment?

Reply:
Through the `security-app`